conans/test/integration/command/create_test.py

```diff
@@ -2,316 +2,285 @@
-                     "test_package/conanfile.py":
-                         GenConanfile().with_test('self.output.info("TESTING!!!")')})
-        client.run("create . --user=lasote --channel=testing")
-        self.assertIn("pkg/0.1@lasote/testing: Generating the package", client.out)
-        self.assertIn("pkg/0.1@lasote/testing (test package): TESTING!!!", client.out)
-
-    def test_create_skip_test_package(self):
-        # Skip the test package stage if explicitly disabled with --test-folder=None
-        # https://github.com/conan-io/conan/issues/2355
-        client = TestClient()
-        client.save({"conanfile.py": GenConanfile().with_name("pkg").with_version("0.1"),
-                     "test_package/conanfile.py":
-                         GenConanfile().with_test('self.output.info("TESTING!!!")')})
-        client.run("create . --user=lasote --channel=testing --test-folder=\"\"")
-        self.assertIn("pkg/0.1@lasote/testing: Generating the package", client.out)
-        self.assertNotIn("TESTING!!!", client.out)
-
-    def test_create_package_requires(self):
-        client = TestClient()
-        client.save({"conanfile.py": GenConanfile()})
-        client.run("create . --name=dep --version=0.1 --user=user --channel=channel")
-        client.run("create . --name=other --version=1.0 --user=user --channel=channel")
-
-        conanfile = GenConanfile().with_require("dep/0.1@user/channel")
-        test_conanfile = """from conan import ConanFile
-class MyPkg(ConanFile):
-    requires = "other/1.0@user/channel"
-    def requirements(self):
-        self.requires(self.tested_reference_str)
-    def build(self):
-        for r in self.requires.values():
-            self.output.info("build() Requires: %s" % str(r.ref))
-        import os
-        for dep in self.dependencies.host.values():
-            self.output.info("build() cpp_info dep: %s" % dep)
-
-    def test(self):
-        pass
-        """
-
-        client.save({"conanfile.py": conanfile,
-                     "test_package/conanfile.py": test_conanfile})
-
-        client.run("create . --name=pkg --version=0.1 --user=lasote --channel=testing")
-
-        self.assertIn("pkg/0.1@lasote/testing (test package): build() "
-                      "Requires: other/1.0@user/channel", client.out)
-        self.assertIn("pkg/0.1@lasote/testing (test package): build() "
-                      "Requires: pkg/0.1@lasote/testing", client.out)
-        self.assertIn("pkg/0.1@lasote/testing (test package): build() cpp_info dep: other",
-                      client.out)
-        self.assertIn("pkg/0.1@lasote/testing (test package): build() cpp_info dep: dep",
-                      client.out)
-        self.assertIn("pkg/0.1@lasote/testing (test package): build() cpp_info dep: pkg",
-                      client.out)
-
-    @pytest.mark.xfail(reason="Legacy conan.conf configuration deprecated")
-    def test_build_folder_handling(self):
```

Comment:
Removed this tests, the rest was semi-automatic unittest -> pytest changes in this file + formatting